runtime/src/juvix/limits.h

```diff
@@ -52,6 +52,12 @@
 
 #define MAX_CLOSURE_SIZE (MAX_FUNCTION_ARGS + CLOSURE_HEAD_SIZE)
 
+#ifdef BITS32
+#define MAX_SMALLINT 1073741823L
```

Comment:
Is it possible to replace the literal by `1<<30 - 1`?

Reply:
Then one can't use it in a 'case', but it's probably fine for this one.

Reply:
I'll change it

Reply:
Done